### PR TITLE
Forms labels

### PR DESCRIPTION
**Pull request by LucRome**

### CODE REVIEW

File: eCourse_backend/courses/urls.py [hunk 1]
```diff
@@ -22,11 +22,13 @@
...
     path('overview/page<int:page>', course_overview, name='course_overview'),
-    # create user iframes
+    # create course iframes
     path('iframes/studentlist/<id>', course_student_list_iframe,
          name='course_studentlist_iframe'),
     # admin: create course
     path('create/', create_course, name="create_course"),
 
     path('edit/<id>', edit_course, name='edit_course'),
+    path('detailed', detailed_course, name='detailed_course'),
```
Comment (phie73) [inline eCourse_backend/courses/urls.py]: should be detailed/<id> but i don't know if we need this still. This was originally created by backend to demonstrate the function. I think you created another template?

File: eCourse_backend/courses/views.py [hunk 2]
```diff
@@ -15,41 +15,50 @@
...
+        # files
+        files = dir()
+        for e in exercise:
+            files[e.id] = Submission.objects.filter(exercise=e.id)
+
+        context = {
+            'lecturer': lecturer,
+            'students': students,
+            'exercise': exercise,
+            'files': files}
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: files have moved to fileexchange/views.py

File: eCourse_backend/courses/views.py [hunk 3]
```diff
@@ -15,41 +15,50 @@
...
+            context)
+
+    # student
+    if (request.user.type == 3):
+        # exercises
+        exercise = Exercise.objects.filter(course_id=id)
+        files = dir()
+        for e in exercise:
+            files[e.id] = Submission.objects.filter(
+                (Q(user=request.user.id) | Q(from_lecturer=1)), exercise=e.id)
```
Comment (phie73) [inline eCourse_backend/courses/views.py]: files have moved to fileexchange/views.py

**Review by phie73 — CHANGES_REQUESTED** [review]
looks good overall, please address the comments

**Comment by LucRome** [comment]
**Warning:**
    * already contains the changes made in design rework
    * only files that are important for this PR are the `forms.py`

**Comment by LucRome** [comment]
Changes: Added the field `labels={...}` to the model forms, and removed one obsolete function (from `users/forms.py`

File: eCourse_backend/templates/home/home_auth.html [hunk 4]
```diff
@@ -15,15 +15,14 @@ <h2 class="navbar-brand ml-auto">
...
-        Dozenten
+        Dozenten)
       {% elif user.type == 3 %}
-        Studenten
+        Studenten)
       {% else %}
-        unbekannte Usergruppe
+        unbekannte Usergruppe)
       {% endif %}
-      )
```
Comment (LucRome) [inline eCourse_backend/templates/home/home_auth.html:26]: Necessary, otherwise it puts a space before ')'
Example: `(Gruppe: Dozenten )`

Bugs me out

**Review by phie73 — APPROVED** [review]
:+1:

**Pull request merged**